src/tenant/views.py

```diff
@@ -76,6 +85,17 @@ def form_valid(self, form):
             owner.last_name = cleaned_data['last_name']
             owner.save()
 
+            # set the owner's username to firstname.lastname (instead of "owner")
+            #
+            # ``generate_unique_username`` method returns a unique username from the combination of strings
+            # present in txts (first argument) iterable, for reference:
+            # https://docs.allauth.org/en/latest/account/advanced.html#creating-and-populating-user-instances
+            user_username(owner, get_adapter().generate_unique_username([
```

Comment:
Is this necessary?  Seems over engineered. On a new deck there are no users yet, except "bytedeck_admin".  Why not just set it to f"{first_name}.{last_name}"?

Reply:
probably..., I will simplify it, if you are requesting